client/lang/src/en.json

```
@@ -9,6 +9,7 @@
     "AssetAdmin.BULK_ACTIONS_DELETE_FOLDER": "These folders contain files which are currently in use, you must move or delete their contents before you can delete the folder.",
     "AssetAdmin.BULK_ACTIONS_DELETE_MULTI_CONFIRM": "There are %s files currently in use, are you sure you want to delete these files?",
     "AssetAdmin.BULK_ACTIONS_DELETE_SINGLE_CONFIRM": "This file is currently in use in %s places, are you sure you want to delete it?",
+    "AssetAdmin.BULK_ACTIONS_DELETE_ITEMS_CONFIRM": "Warning: You're about to delete %s file(s) which may be used in your site's content. Please carefully check the file usage to ensure they are removed from the content areas prior to deleting them, otherwise they will appear as broken links.",
```

The message is displayed in an obnoxious modal. Is it not clear enough it's a warning?
```suggestion
    "AssetAdmin.BULK_ACTIONS_DELETE_ITEMS_CONFIRM": "You're about to delete %s file(s) which may be used in your site's content. Carefully check the file usage to ensure they are removed from the content areas prior to deleting them, otherwise they will appear as broken links.",
```

---

@clarkepaul ?

---

Was a bit torn on this one as the "waring:" text isn't best practice but the circumstances of not reading the text could be pretty damaging. It was only there because we are being a little lazy with the visuals communicating its importance e.g. no warning icon. I've been swayed towards removing it as Max has a good point, the whole modal acts as the warning. 

FYI its also best practice to avoid using the word "please" in UIs so that is good it was removed.

---

As per @clarkepaul's comment above, this still needs actioning

---

This was previously actioned